monitoring/dr/alerts.test.yaml

```
@@ -0,0 +1,88 @@
+# These tests are meant to go hand-in-hand with the rendered alert rule.
+# Use github.com/scality/action-prom-render-test@python-renderer python module
+#
+# Render the alerts file with
+# gen-alert render alerts.yaml
+evaluation_interval: 1m
+rule_files:
+  - alerts.rendered.yaml
+
+tests:
+  - interval: 1m
+    input_series:
+      - series: up{drSinkInstance="", job="artesca-data-dr-source-base-queue-connector-metrics", namespace="zenko"}
```

such serie does not exist, we have either (on the source):
```
up{job="artesca-data-dr-source-base-queue-connector-metrics", namespace="zenko"}
```

or (scraped via federation from the sink):
```
up{drSinkInstance="artesca-data-dr", job="artesca-data-dr-source-base-queue-connector-metrics", namespace="zenko"}
```

```suggestion
      - series: up{job="artesca-data-dr-source-base-queue-connector-metrics", namespace="zenko"}
```

(here, you define actual metrics series ; which is not exactly the same as the _selectors_ we use in alerts/dashboards to query metrics)

---

(same for followup metrics)

---

Done here : https://github.com/scality/Zenko/pull/2130/commits/e382a8492d1046e4529a23ecd4fc8c219b7d3f14